### PR TITLE
Adding User Controller changes along with changes to the html pages f…

### PR DESCRIPTION
**Pull request by Nakirendral**
…or index and new

### CODE REVIEW

File: app/controllers/lists_controller.rb [hunk 1]
```diff
@@ -2,7 +2,11 @@ class ListsController < ApplicationController
 #before_action :find_list, only: [:show, :update, :edit, :destroy]
 
   def index
-    @lists = List.all.order("created_at DESC")
+    @lists = List.all
```
Comment (FifthSurprise) [inline app/controllers/lists_controller.rb:5]: Out of curiosity, why did you remove this order?

File: app/views/lists/index.html.erb [hunk 2]
```diff
@@ -3,16 +3,16 @@
 <table>
   <thead> 
     <tr>
-      <th>Title</th>
-      <th>Content</th>
+      <th>Title </th>
```
Comment (FifthSurprise) [inline app/views/lists/index.html.erb:6]: Extra space?

File: app/views/lists/index.html.erb [hunk 3]
```diff
@@ -3,16 +3,16 @@
...
+      <th>Content </th>
```
Comment (FifthSurprise) [inline app/views/lists/index.html.erb:7]: And an extra space here too.

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+<div class="section">
+  <p>
+    <%= @user.first_name %>
+    <br>
+    <%= @user.last_name %>
+    <br>
+    <%= @user.email %>
+    <br>
+  </p> 
+</div>
```
Comment (FifthSurprise) [inline app/views/users/show.html.erb:12]: Newline

File: app/controllers/lists_controller.rb [hunk 5]
```diff
@@ -12,10 +16,10 @@ def new
   def create
     @list = List.new(list_params)
 
-    if @post.save
+    if @list.save
       redirect_to @list
-    else
```
Comment (FifthSurprise) [inline app/controllers/lists_controller.rb:17]: Why did you comment out this rendering of the new page if `@list.save` fails?

**Review by FifthSurprise — CHANGES_REQUESTED** [review]
Please write tests for the things you are doing.  You can use the `lists_controller.spec` as a baseline.  For further depth, you can use [Relish](https://relishapp.com/rspec).  While I won't put a high bar on what kind of tests you write, it is important that you _do_ write tests for your code if you can.

In addition, it looks like you tried to make both user controller changes and list related changes in the same PR.  In the future, try to split those two into different PR's.  Also, I'm not sure if we need to do a lot with the User's class right now.  In the future we'll probably use something like [Devise](https://github.com/heartcombo/devise) to manage all of that.  It's good that you're thinking about it but in the short term, we're adding a lot of boilerplate code we may not need.

File: app/controllers/lists_controller.rb (hunk 1 above)
Comment (Nakirendral) [reply app/controllers/lists_controller.rb:5]: I was afraid that it was a reason for my inability to create List items. I will put it back in since that didn't seem to matter.
